aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -237,6 +237,8 @@ See the [Visual Studio publish profiles for ASP.NET Core app deployment](xref:ho
 
 Use any of several methods to move the app to the hosting system, such as manual copy, Xcopy, Robocopy, or PowerShell.
 
+For more information on ASP.NET Core deployment to IIS, see the linked content in the [Deployment resources for IIS administrators](#deployment-resources-for-iis-administrators) section at the end of this topic.
```

Comment:
I recommend removing "at the end of this topic". You're linking to it, so that part seems unnecessary.

Reply:
I often prefer to use "in this topic" language with links that are bookmarks (and I'm not the only one; others have done this on the repo) because the link text doesn't indicate where the links goes. I want to reader to know two things by doing this ...

1. You're not going to be navigated away someplace else. When I'm reading a topic, I sometimes don't click a link (even when I'm interested in the linked content) because I'm concentrating on the topic and don't want to be thrown someplace else at that time.
1. More importantly ... If the reader knows that the content is coming up in the same topic, there's really no need to click that link when reading a topic top-to-bottom. This tells them, 'you're gonna get to this content in a minute anyway.'

However, I do have "section." That's not too bad. Even if @Rick-Anderson prefers we drop the "at the end" bit, I think we should at least leave "section." I'd prefer to leave it as written tho. I want to make it clear to the reader that they don't need to click that link if they're reading the whole topic.

@Rick-Anderson @rachelappel Thoughts?

Reply:
I prefer:

For more information on ASP.NET Core deployment to IIS, see ~the linked content in the~ [Deployment resources for IIS administrators](#deployment-resources-for-iis-administrators) ~section at the end of this topic.~

I see @guardrex  point, so I don't have strong feelings about it. But they'll find quickly enough it's  in the same page. 

On Maybe as a comprimise:

I perfer:

For more information on ASP.NET Core deployment to IIS, see ~the linked content in the~ [Deployment resources for IIS administrators](#deployment-resources-for-iis-administrators) section ~at the end of this topic.~